package.json

```diff
@@ -3,6 +3,7 @@
   "version": "1.13.0",
   "description": "Stripes Command Line Interface",
   "repository": "https://github.com/folio-org/stripes-cli",
+  "uiTemplateRepository": "https://github.com/folio-org/ui-app-template.git",
```

Comment:
The CLI hasn't used `package.json` for its own configuration (yet).  It does support a [.stripesclirc](https://github.com/folio-org/stripes-cli/blob/2c9dff2d3f2522535e2626bdceb814c339edfb43/lib/cli/config.js#L14) config and also has inventory concept in [libs/environment/inventory.js](https://github.com/folio-org/stripes-cli/blob/master/lib/environment/inventory.js).  While `package.json` is certainly functional here, I hesitate to introduce a third pattern into the mix. Doing so reduces clarity (in what is perhaps not already clear) of where things like this would go next time around.

In my opinion, `.stripesclirc` would be ideal, as it would naturally allow for user-defined values, enabling user-defined template repos!  (I could see that being pretty cool for a university or dev team that wants to use their own boilerplate).   However, the CLI doesn't yet have a notion of _default_ `.stripesclirc` values (when no file is present) and that's probably beyond the scope of this PR.

So to keep it simple and reuse an existing pattern, exporting the URL from inventory.js may be the easiest option.  It also seems fitting that templates could be part of this "inventory."

```
const templates = {
 uiApp: "https://github.com/folio-org/ui-app-template.git"
}

module.exports = {
  uiModules,
  stripesModules,
  templates,
  // ...
}
```